worker/appm/conversion/version.go

```diff
@@ -196,25 +196,6 @@ func createEnv(as *v1.AppService, dbmanager db.Manager) (*[]corev1.EnvVar, error
 		if es != nil {
 			envsAll = append(envsAll, es...)
 		}
-
```

Comment:
Remove `DEPEND_SERVICE ` `DEPEND_SERVICE_COUNT ` env, It's not reasonable. These are two important variables